recipes/alphatwirl/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "alphatwirl" %}
+{% set version = "0.23.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 92bd73cb45646ad7a58b039b12e4566b4fc337406fb142c4c8b3ed1170c32b93
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - atpbar >= 1.0.2
```

Comment:
These should be `package >=X.Y.Z` instead of `package >= X.Y.Z`. The space between the `>=` and the version breaks the CI that makes the feedstock after the PR is merged.

Reply:
I see. Thank you.